### ISSUE TITLE
[Enemy Family] Slackjaws

### ISSUE THREAD

**Issue by Floofball** [ProbablyTez/Agony#403]
Slackjaws, at first glance, appear as slow moving Gapers with no eyes (similar to Blue Gapers). However, as soon as Isaac gets close, they open their jaws and scream loudly, disorienting Isaac and reversing his controls for a brief period of time. After screaming, Slackjaws' jaws will remain slacked and instead simply chase Isaac at about the speed of a Gazing Globin.

Example: http://imgur.com/a/wFSYF

Other types include the Slacker (found in the Catacombs and cough slowing spores), Wrapjaw (found in the Caves onward with much more powerful scream and turn into Boneys after screaming), Brackjaw (found in Flooded Caves and cough large, damaging water puddle w/ 2 Drowned Chargers, turn into Drowned Boneys), Sackjaw (have a sack over their head and inflate, release many coin shots on death), and the Blackjaw (found in Sheol and shoot a curving Brimstone laser).

**Tez's TL;DR** : Edited for Tez's suggestions by Floof

>  - **Slacker family** : 
>     - **Initial weaker AI**
>     - **Triggers one time only attack on proximity**
>     - **Switches to stronger AI afterward**

- **Slackjaw** : 
   - Weak AI : Slowly chases player
   - Attack : Disorient player
   - Strong AI :  Faster chase
- **Slacker** : 
   - Weak AI : Slowly chase player
   - Attack : Slowing spores, pushes player back
   - Strong AI : Slower, more HP, slowing & damaging green creep
- **Wrapjaw** : 
   - Weak AI : Slowy chase player
   - Attack : Pushes, disorients & fears (can't shoot) player & enemies
   - Strong AI : Turn to Boney
- **Brackjaw** : 
   - Weak AI : Slowy chase player
   - Attack : Tear Puddle w/ 2 Drowned Chargers
   - Strong AI : Turn to Drowned Boney
- **Sackjaw** : 
   - Weak AI : Slowly chase player
   - Attack : Inflates, increased hitbox 
   - Strong AI : Faster chase, explodes into coin shots on death
- **Slackmaw** : 
   - Weak AI : Slowly chase player
   - Attack : Curved Brimstone w/ mass fear
   - Strong AI : Leaping chase

**Comment by ProbablyTez** [comment]
I really really like the concept, however I feel the different kind of Slackjaws are too similar.
- Slacker's gas is mechanically the same as slowing creep. Also, Catacombs have a minor poisoning gas theme, this might be more appropriate 😉 
- Suggestion for Wrapjaw's attack : Way more powerful scream than Slakjaws', pushes the player away, disorienting him and fearing other enemies briefly
- Brackjaw actually feel pretty weak and not interesting, how about hush like tear tornado instead ? Since it's a one time attack it can be pretty strong
- Here's an idea for sackjaws : The sack muffles their scream, instead of screaming the sack inflates, and on death, releases a burst of coins
- Maybe blackjaws should have another strong AI

Other than that though, really like the concept, good job!

**Comment by Anonymous-Koala** [comment]
I like the concepts. I'd also revisit it more closely once I'm done with my Enemy and Boss guideline sheet, so we have a form to judge them by.
